eCourse_backend/requirements.txt

```diff
@@ -1,3 +1,4 @@
 Django==3.1.7
 django-extensions
-django-widget-tweaks
\ No newline at end of file
+django-widget-tweaks
+python-magic==0.4.22
```

Comment:
We have a small problem with python-magic. It needs a shared library which has to be installed separately. See https://pypi.org/project/python-magic/

We could a) also depend on the binary version for windows or b) just say it only works on linux

Reply:
i would go for b) but beside the fact that we have way to many people in our team using windows i should propaply go for a) or a total different solution

Reply:
found a solution without magic, now it is hardcoded to pdf but I think it is fine because we only allow to upload pdfs to the server